docs/source/guide/decommission_minio.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Decommission minio
+short: Decommission minio
+meta_title: Decommission minio
+---
+
+> This is too dangerous action. You may lose all your data uploaded using direct file upload.
+> If you're not confident with the following steps, you can request help from the Heartex Team.
+
```

Comment:
Would you like to replace lines #7 and #8 with a warning?

 If yes, replace lines #7 and #8 with the following: 
> WARNING: Decommissioning Minio is a hazardous action because you may lose all your data uploaded using direct file upload. For more information or support, contact the Heartex team.

Reply:
Line #9 will be a line break. 

Line #10 
To decommission Minio: 

Line #11 will be a line break. 

Line #12 
## Configure Persistent Storage